src/Atmos/Model/tendencies_energy.jl

```diff
@@ -0,0 +1,33 @@
+##### Energy tendencies
+
+#####
+##### First order fluxes
+#####
+
+function tendency(::Advect{Energy}, m, state, aux, t, ts, direction)
```

Comment:
I don't think we should use `tendency` here, since it isn't applied directly as a tendency (its divergence is). How about:
```suggestion
function flux(::Advect{Energy}, m, state, aux, t, ts, direction)
```